control-plane-operator/controllers/hostedcontrolplane/autoscaler/v2/reconcile.go

```diff
@@ -0,0 +1,111 @@
+package autoscaler
+
```

Comment:
how about instead we have the folder path as v2/component name so it's visible at a glance all the migrated components

Reply:
yeah makes sense